### PR TITLE
Styles

### PR DESCRIPTION
**Pull request by ReAliens**
in this project we did:
- add A Navigation bar with three sections 
- add date and time 
- add a footer 
- update styles and js for making a website as a single page application

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,17 +8,32 @@
...
+      <p>
+        <span> Authors</span>: Ersoy,Reda,Didi,<br />
+        <span> Organisation</span>: microverse,<br />
+        <span> Website</span> :microverse.org, <br />
+        All rights reserved
+      </p>
+    </section>
+    <section class="dis-none" id="booklistsection">
+      <h2 class="title2">Your book list</h2>
       <div class="books-wrapper" id="books-wrapper"></div>
```
Comment (rloterh) [inline index.html:34]: - [ ] The styling of your _books list rows_ background color does not conform to the database table alternate rows color, as can be seen in the original wireframe below. Kindly ensure to improve upon this aspect.

Your design:
image

**The wireframe design:**
image

File: index.html [hunk 2]
```diff
@@ -37,6 +52,9 @@ <h2>Insert Books</h2>
         <button type="submit" id="add-btn">Add</button>
```
Comment (rloterh) [inline index.html]: The _add button_ is not aligned appropriately. Please note that we actually want our design to as much possible follow the [wireframe template](https://github.com/microverseinc/curriculum-javascript/blob/bb245ca8a9d3bb04ebefbbd45e800764e1fc5f74/books/images/awesome_books_core_elements.png) Kindly improve this aspect by ensuring that
- [ ] The _add button_ is **aligned to the right** with the form input, as can be seen in the wireframe design.

**Your design:**
image


**The wireframe design:**
image

File: app.js [hunk 3]
```diff
@@ -90,3 +90,48 @@ document.querySelector('.books-wrapper').addEventListener('click', (e) => {
   Book.deleteBook(e.target);
   Book.removeBooks(e.target.id);
 });
```
Comment (rloterh) [inline app.js:92]: - [ ] As specified, kindly ensure to use the Luxon library to implement the date found below the navigation bar.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Team, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- Descriptive Pull Request
- Nice design
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi Team, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**